lib/ukfallocbuddy/tests/test_ukfallocbuddy.c

```diff
@@ -0,0 +1,217 @@
+#include <uk/test.h>
+#include <uk/fallocbuddy.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+struct test_ukfallocbuddy {
+
+	struct uk_falloc *fa;
+	__paddr_t start;
+	__sz len;
+	unsigned long pages;
+	__sz fa_meta_size;
+	void *start_zone;
+	void *fa_meta;
+};
+
+
+static struct uk_falloc *init_new_fb_alloc(void *fa_meta, void *start_zone,
+	unsigned long len, __paddr_t *new_start, __sz fa_meta_size,
+	unsigned long *pages)
```

Comment:
Parameters should be aligned. Since this will not allow you to put more than one parameter per line it is fine to move the return value to the previous line.
```suggestion
static struct uk_falloc *
init_new_fb_alloc(void *fa_meta, void *start_zone, unsigned long len, 
		  __paddr_t *new_start, __sz fa_meta_size, unsigned long *pages)
```